test/libsolidity/syntaxTests/globalFunctions/blobhash_var_shadow_warning.sol

```diff
@@ -0,0 +1,9 @@
+contract C
+{
+    function f() public pure returns (bool) {
+        bool blobhash = true;
+        return blobhash;
+    }
+}
+// ----
+// Warning 2319: (67-80): This declaration shadows a builtin symbol.
```

Comment:
I see no `EVMVersion` here so this warning is issued on all versions? `blobhash()` exists only on `>=cancun` so we should warn only there.